tests/unit/protocol.tcl

```diff
@@ -135,6 +135,30 @@ start_server {tags {"protocol network"}} {
         assert_equal [r read] {a}
     }
 
+    test "bulk reply protocol" {
+        # int encoding
+        r set crlf 2
+        assert_equal [r rawread 5] "+OK\r\n"
+        r get crlf
+        assert_equal [r rawread 7] "\$1\r\n2\r\n"
+        r set crlf 2222
+        assert_equal [r rawread 5] "+OK\r\n"
+        r get crlf
+        assert_equal [r rawread 10] "\$4\r\n2222\r\n"
+        r set crlf 2222222222222222
+        assert_equal [r rawread 5] "+OK\r\n"
+        r get crlf
+        assert_equal [r rawread 23] "\$16\r\n2222222222222222\r\n"
```

Comment:
what about testing the min long long (-9223372036854775808), max long long (-9223372036854775808)  and an ordinary number?

Reply:
and it's better to use `object encoding crlf` to check if the encoding is correctly.